README.md

```diff
@@ -44,11 +44,7 @@ We recommend that you set the *Dependency Rule* to *Up to Next Major Version*. W
 
 Before you can use the Customer.io SDK, you need to initialize it. Any calls that you make to the SDK before you initialize it are ignored. 
 
-There are two ways to initialize the SDK. The method you use depends on how you want to use the SDK:
-
-1. Singleton, shared instance (the quick and easy way):
-
-When you use the shared instance, you don't need to manage your own instances of Customer.io SDK classes. To get started, initialize the SDK in the `AppDelegate` `application(_ application: didFinishLaunchingWithOptions)` function: 
+To get started, initialize the SDK in the `AppDelegate` `application(_ application: didFinishLaunchingWithOptions)` function: 
```

Comment:
This change to the README means we need to make an update to the public facing docs for the SDK.

Reply:
[Confirmed](https://customer.io/docs/sdk/ios/getting-started/#initialize-the-sdk)